### ISSUE TITLE
If only one assay is present and is nameless, the assay column is not printed

### ISSUE THREAD

**Issue by stemangiola** [stemangiola/tidySummarizedExperiment#52]
@mblue9 this is the issue you mentioned

If only one assay is present and is nameless, the assay column is not printed. We could create a column with a default name

which name could be best (?)

`assay_1`, `assay_2`

**Comment by mblue9** [comment]
If it's not too much work to add you could use  `assay_1`

**Comment by stemangiola** [comment]
Yes it is not extremely trivial as it would seem. I am not sure the timeline it's compatible with the workshop, but I'll do my best ;)
